models/activityfeed.js

```diff
@@ -0,0 +1,19 @@
+let mongoose = require('mongoose');
+
+let activityFeedSchema = mongoose.Schema({
+  owner: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
```

Comment:
Having a User object here likely makes this more of a NewsFeed, this goes back again to my non-relational database inexperience though, unsure if this reference id should exist on the parent or child or both.